api/server.js

```diff
@@ -1,3 +1,92 @@
 // BUILD YOUR SERVER HERE
+const express = require('express')
+const Users = require(`./users/model`)
+const server = express()
 
-module.exports = {}; // EXPORT YOUR SERVER instead of {}
+server.use(express.json())
+
+//Endpoints:
+// GET requests:
+server.get(`/api/users`, async (req, res) => {
+   try {
+      const users = await Users.find()
+      res.status(200).json(users)
+   } catch (err) {
+      res.status(500).json({ message: "The users information could not be retrieved" })
+   }
+})
+
+server.get(`/api/users/:id`, async (req, res) => {
+   try {
+      const { id } = req.params
+      const userId = await Users.findById(id)
```

Comment:
nit: Is the data coming back from the db a userId? Would there be a better word to describe the data